RMSD.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+### Authors:
+
+# Daisy Y. Kyu
+# Nam Thi
+# Victoria Lim (limvt@uci.edu)
+# Caitlin C. Bannan (bannanc@uci.edu)
+
+### Description: This Python script 
+import os
+import openeye.oechem as oechem
+
+#-------------------- Functions---------------------#
+
+
+def RMSD(ffRef, ffList, molSet, molName):
+    """
+    From a set of reference molecules and query molecules, the RMSD values are
+     computed and wrote into a file.  
+   
+    Parameters
+    ---------
+    ffRef: str - the reference forcefield
+    ffList: str - the query forcefield
+    molSet: str - set of molecule
+    molName: str - name of the molecules
+   
+    Returns
+    -------
+    RMSD values for each molecules with the defined ff
+ 
+    """
+    # Set up an error file to record molecule does not exist
+    errFile = open('rmd_logfile.txt','a')
+
+    # Set up a file that catch negative values
+    nValue = open('negative_value.txt','a') 
+  
+    # Set up a log file for RMSD
+    logFile = open('RMSD.txt', 'a')
+   
+ 
+    # open reference molecule
+    refFile = ("/work/cluster/nthi/ForceField-Comparison/%s/%s/%s" % (molSet, ffRef, molName) )
+    ifsRef = oechem.oemolistream(refFile)
+    print "Opening reference molecule:", refFile
+
+    # check if the file exist
+    if not ifsRef.open(refFile):
+        print ("Unable to locate %s. Skipping." % refFile)
+
+    # open query molecule
+    queryFile = ("/work/cluster/nthi/ForceField-Comparison/%s/%s/%s" % (molSet, ffList, molName) )
+    print "Opening query molecule: ", queryFile   
+    if os.path.exists(queryFile):
+        ifsQuery = oechem.oemolistream(queryFile)
+   
+        # set flavor for the input file
+        flavor = oechem.OEIFlavor_Generic_Default | oechem.OEIFlavor_MOL2_Default | oechem.OEIFlavor_MOL2_Forcefield
+        ifsRef.SetFlavor(oechem.OEFormat_MOL2, flavor)
+        ifsQuery.SetFlavor(oechem.OEFormat_MOL2, flavor)
+   
+        # create "blank" object
+        rmol = oechem.OEGraphMol()
+        qmol = oechem.OEGraphMol()
+   
+        # load molecule from files
+        oechem.OEReadMolecule(ifsRef,rmol)
+        oechem.OEReadMolecule(ifsQuery,qmol)
+       
+        # rmsd calculate
+        rms = oechem.OERMSD(rmol,qmol, True, True , True)
+        molNames = rmol.GetTitle()
+        logFile.write("%s\t%s\t%s\t%.3e\n" % (molNames, ffRef,ffList, rms) )
+       
+        # if rms = -1 write into log file
```

Comment:
Instead of commenting `if rms = -1` we can put something more descriptive, like "different smiles strings detected"